### ISSUE TITLE
Y-Axis Specific Tensioner

### ISSUE THREAD

**Issue by francistheodorecatte** [pessimism/TronxyX1#2]
The tensioner you linked for the X and Y axes is meant just for the X-axis. There's a design for the Y-axis with the same step down as the stock acrylic fixed idler that's meant for Creality and Ender printers, but fits the TronXY cantilever printers too thanks to the identical belt routing design and use of 2040 profile for the Y-axis rail.

Example link on AliExpress (this listing sells both X and Y axes tensioners):
https://www.aliexpress.com/item/33049736568.html

The linked tensioner doesn't have a toothed pulley, but imo that's unnecessary (and just adds friction). The main problem with the stock idler setup is the low quality bearing, and only being affixed on one side. Ball bearings don't like shear forces, especially cheap ones.

Beyond the tensioner/idler design, there's also other sources of belt tension problems. The stock belts are pretty low quality, and varied in tension along their length on my printer; I ended up just buying better belts to completely solve the belt tension issue. I also had to swap both stock GT2-16 stepper pulleys for better ones, since they had noticeable (to my eye!) runout.

Thanks for the Marlin profile, btw!

**Comment by pessimism** [comment]
You're right, I didn't realize until months later doing a large edge to edge print and smacking into the tensioner.  I ended up modelling and printing this guy https://github.com/pessimism/TronxyX1/blob/master/Models/Parts/bracket%20for%20tensioner/bracket.stl for one side of the Y tensioner.  I still lose a few MM of build area from the X tensioner due to other mods and need to remodel both sides of it for full clearance.  I'm experimenting with printed toothed idlers/pulleys for my other printer projects and still working to get the tooth profile and roundness dialed in to grip the cheap belts properly.  Half the struggle with this stuff is poor part quality and part sourcing in general...
